recipe/meta.yaml

```diff
@@ -1,41 +1,43 @@
 {% set name = "onnx" %}
-{% set version = "1.13.1" %}
+{% set version = "1.16.0" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
   url: https://github.com/{{ name }}/{{ name }}/archive/v{{ version }}.tar.gz
-  sha256: 090d3e10ec662a98a2a72f1bf053f793efc645824f0d4b779e0ce47468a0890e
+  sha256: 0ce153e26ce2c00afca01c331a447d86fbf21b166b640551fe04258b4acfc6a4
 
 build:
   number: 0
   entry_points:
     - check-model = onnx.bin.checker:check_model
     - check-node = onnx.bin.checker:check_node
     - backend-test-tools = onnx.backend.test.cmd_tools:main
+  missing_dso_whitelist:  # [s390x]
+    - '$RPATH/ld64.so.1'  # [s390x]
 
 requirements:
   build:
-    - {{ compiler('c') }}                     
-    - {{ compiler('cxx') }}                   
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
     - cmake
-    - make                                    # [not win]
-    - libprotobuf
+    - make          # [not win]
+    - libprotobuf {{ libprotobuf }}
   host:
     - python
     - pip
     - setuptools
     - wheel
-    - pybind11 2.10.4
+    - pybind11 2.12.0
     # required >=3.20.2 but we have only 3.20.3 with python 3.11 support
     - protobuf 3.20.3
-    - libprotobuf
+    - libprotobuf {{ libprotobuf }}
     - pytest-runner 6.0.0
-    - numpy   1.19  # [py<310]
-    - numpy   1.21  # [py==310]
-    - numpy   1.23  # [py>=311]
+    - numpy   1.21  # [py<310]
```

Comment:
can we just do `numpy {{ numpy }}`?

Reply:
Because 3.10 min is set to numpy 1.23: https://github.com/onnx/onnx/blob/v1.16.0/requirements-min.txt